molecule/cookiecutter/scenario/driver/linode/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -0,0 +1,17 @@
+********************************
+Linode driver installation guide
+********************************
+
+Requirements
+============
+
+* ``LINODE_API_KEY`` exposed in your environment
+* ``linode-python`` Python dependency installed
+  * Supports Python 2.7+ but is not Python 3.X compatible.
+
+Install
+=======
+
+.. code-block:: bash
+
+    $ pip install linode-python
```

Comment:
maybe?
```suggestion
    $ pip install molecule[linode]
```

OTOH this can be done separately, fixing all other drivres as well

Reply:
Going to leave this for now.

Reply:
I think you should add it since all other drivers in master have their extras entries now.